Expensify/App
# add step counter to VBA flow

### Details

A step counter is added in the VBA flow.

### Fixed Issues

$ https://github.com/Expensify/App/issues/5074
$ https://github.com/Expensify/Expensify/issues/177670

### Tests

1. Connect to the local environment following these [instructions](https://stackoverflow.com/c/expensify/questions/7699/7700).
2. Login with a test account.
3. Click on the "+" button -> "New workspace" -> "Get Started" or by entering /bank-account into the URL (more details [here](https://stackoverflow.com/c/expensify/questions/10699)).
4. Follow the steps of the VBA flow and verify the step counter below the header of each step.

### QA Steps

1. Click on the "+" button -> "New workspace" -> "Get Started" or by entering /bank-account into the URL (more details [here](https://stackoverflow.com/c/expensify/questions/10699)).
2. Follow the steps of the VBA flow and verify the step counter below the header of each step (more details of adding a bank account in staging [here](https://stackoverflow.com/c/expensify/questions/342).

### Tested On


### Screenshots


#### Web

<img width="1187" alt="" src="https://user-images.githubusercontent.com/6829422/133833294-3c0b4dcb-39e8-42b0-97eb-0dfdd20f70e7.png">
<img width="1187" alt="" src="https://user-images.githubusercontent.com/6829422/133346645-623bef6d-3193-47e4-9df9-6eb1d025706a.png">
<img width="1187" alt="" src="https://user-images.githubusercontent.com/6829422/133346640-16eb4c19-e77f-47b1-9ceb-43eec1b264d1.png">
<img width="1187" alt="" src="https://user-images.githubusercontent.com/6829422/133346635-12e7e759-0dcd-4ad4-83c7-368c7a561010.png">
<img width="1187" alt="" src="https://user-images.githubusercontent.com/6829422/133346633-6358d7bf-6184-4cee-821f-a76104bcd10c.png">
<img width="1187" alt="" src="https://user-images.githubusercontent.com/6829422/133346629-0f77b160-ddfe-45a7-bea8-df502400b885.png">

#### Mobile Web

<img width="300" alt="" src="https://user-images.githubusercontent.com/6829422/133833335-98b06f49-027e-411a-9381-9ae6dfd76804.png">
<img width="300" alt="" src="https://user-images.githubusercontent.com/6829422/133347294-b2ce7e63-86be-4b2b-a6ca-72a3534f35a1.png">
<img width="300" alt="" src="https://user-images.githubusercontent.com/6829422/133347291-3350af20-f030-496d-98da-d473de37ecd5.png">
<img width="300" alt="" src="https://user-images.githubusercontent.com/6829422/133347290-efea9842-8ea7-4f19-b92e-1d6400333691.png">
<img width="300" alt="" src="https://user-images.githubusercontent.com/6829422/133347285-fbb9d813-f514-49a7-a094-9183e7b86aaa.png">
<img width="300" alt="" src="https://user-images.githubusercontent.com/6829422/133347280-d7d6c722-3e53-4063-9d85-e40ab100e942.png">

#### Desktop

<img width="750" alt="Screen Shot 2021-09-17 at 12 08 14" src="https://user-images.githubusercontent.com/6829422/133833393-dbf1b923-2e85-4055-9612-a9a719865ca7.png">
<img width="750" alt="Screen Shot 2021-09-14 at 18 42 01" src="https://user-images.githubusercontent.com/6829422/133349028-0285e7fe-56f4-401d-bbe1-e50eaf46907b.png">
<img width="750" alt="Screen Shot 2021-09-14 at 18 44 21" src="https://user-images.githubusercontent.com/6829422/133349032-5dfb3790-3d1b-48f6-90dd-a2dc15766323.png">
<img width="750" alt="Screen Shot 2021-09-14 at 18 43 54" src="https://user-images.githubusercontent.com/6829422/133349030-97c43f28-0311-4e30-8c16-180cbc7feb59.png">
<img width="750" alt="Screen Shot 2021-09-14 at 18 50 03" src="https://user-images.githubusercontent.com/6829422/133349035-e0596e73-a728-4a8f-9783-e757f0f0df55.png">
<img width="750" alt="Screen Shot 2021-09-14 at 18 51 57" src="https://user-images.githubusercontent.com/6829422/133349037-d50a31bb-b696-42b0-8f84-816a4b6e0f26.png">
<img width="750" alt="Screen Shot 2021-09-14 at 18 52 19" src="https://user-images.githubusercontent.com/6829422/133349038-1c48dc59-0b23-47d7-a1a0-98ac12842797.png">

#### iOS
<img width="300" alt="" src="https://user-images.githubusercontent.com/6829422/133833444-f70c3217-c5d5-40e5-9be6-c336898bda7c.png">
<img width="300" alt="" src="https://user-images.githubusercontent.com/6829422/133518702-b8135466-ba79-4195-95f1-f5712ce99714.png">
<img width="300" alt="" src="https://user-images.githubusercontent.com/6829422/133518704-8536f6d9-9c34-4dee-9168-791d94100916.png">
<img width="300" alt="" src="https://user-images.githubusercontent.com/6829422/133518705-e9281aa7-5bc8-48ff-8567-34526b35d7f3.png">
<img width="300" alt="" src="https://user-images.githubusercontent.com/6829422/133518706-70d1b5fb-a743-431a-87f6-8ab959a67e70.png">
<img width="300" alt="" src="https://user-images.githubusercontent.com/6829422/133518708-35ca5b9a-0711-42e3-9c46-afb46fc07cf3.png">

#### Android

<img width="300" alt="" src="https://user-images.githubusercontent.com/6829422/133833495-5ab30508-fed1-440f-8380-9228d9aec61d.png">
<img width="300" alt="" src="https://user-images.githubusercontent.com/6829422/133657672-34386d71-de0a-487d-b9fa-ad7ec7af114b.png">
<img width="300" alt="" src="https://user-images.githubusercontent.com/6829422/133657676-90fdd3fb-eb10-4539-abab-cc42e732be5a.png">
<img width="300" alt="" src="https://user-images.githubusercontent.com/6829422/133657679-0aa0a953-0059-4615-9401-5100b3a7e037.png">
<img width="300" alt="" src="https://user-images.githubusercontent.com/6829422/133657682-8d359d83-5a1a-4b23-acd1-a694db4a4913.png">
<img width="300" alt="" src="https://user-images.githubusercontent.com/6829422/133657687-da4e132c-3c6e-4aa2-8c39-4389df5b06d2.png">
